lib/realtime_web/channels/realtime_channel.ex

```diff
@@ -502,7 +439,9 @@ defmodule RealtimeWeb.RealtimeChannel do
   end
 
   defp confirm_token(%{assigns: %{jwt_secret: jwt_secret, access_token: access_token} = assigns}) do
-    with jwt_secret_dec <- decrypt_jwt_secret(jwt_secret),
+    secure_key = Application.get_env(:realtime, :db_enc_key)
```

Comment:
Nitpick: wouldn't be better to raise if the secure key is not configured?

```suggestion
    secure_key = Application.fetch_env!(:realtime, :db_enc_key)
```

Reply:
oh great call! 🙏